packages/paste-website/src/pages/components/heading/index.mdx

```diff
@@ -185,6 +186,15 @@ It is important to not skip one or more heading levels as it is common for scree
 
 Headings should be used for page headings or content subsections. Headings are broken into six levels, where h1 is the highest level and h6 is the lowest. Avoid skipping heading levels. Use the next lowest heading level to maintain a coherent document outline.
 
+<DoDont>
+  <Do title="Do" body="Use sentence case in headings.">
```

Comment:
Have you tried the `center` prop on these? Might look neater

Reply:
that looks way better, updated!